images/windowsserver2022/README.md

```diff
@@ -0,0 +1,88 @@
+# Windows 2022 Server Overview
+
+Installing a Windows Operating System from a mounted iso as part of your Packer build requires an answer file. 
+
+For more information on how to create and customize an answer file, see [answer files](https://docs.microsoft.com/en-us/windows-hardware/manufacture/desktop/update-windows-settings-and-scripts-create-your-own-answer-file-sxs?view=windows-11) and [unattend](https://docs.microsoft.com/en-us/windows-hardware/customize/desktop/unattend/) docs.
+
+In addition to the OS install, WinRM is enabled and is used as the Communicator for Packer.
```

Comment:
Pardon my nits - just lining up a few capitalization items with whether they're acronyms/always capitalized or not

```suggestion
In addition to the OS install, WinRM is enabled and is used as the communicator for Packer.
```

Reply:
No worries. Thanks for the input!